### PR TITLE
Scaffold page contact

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

**Comment by mixcas** [comment]
It's fine just move the map script to a file or somthing

File: js/main.js [hunk 1]
```diff
@@ -85,6 +89,83 @@ Site.Gallery = {
...
+    ];
+
+    // Info Window Content
+    var infoWindowContent = [
+      ['<div class="info_content">' + WP.mapData.London + '</div>'],
+      ['<div class="info_content">' + WP.mapData.Scotland + '</div>'],
+    ];
+
+    // Display multiple markers on a map
+    var infoWindow = new google.maps.InfoWindow(), marker, i;
```
Comment (mixcas) [inline js/main.js:136]: does this work? @varffs

Comment (varffs) [inline js/main.js:136]: yh. not for you?

Comment (mixcas) [inline js/main.js:136]: everything runs fine. how does this line work?

Comment (varffs) [inline js/main.js:136]: don't actually know its existing code

Comment (mixcas) [inline js/main.js:136]: WTF 
k

**Pull request merged**